wled00/wled.cpp

```diff
@@ -34,6 +34,11 @@ void WLED::reset()
 
 void WLED::loop()
 {
+  if (otaInProgress)
+  {
+    // stop the loop while ota in progress
+    return;
```

Comment:
I suspect it's this change that is causing you the issues with the MQTT clients connection state.

This feels too heavy handed

Reply:
I concur.

Reply:
This is unrelated to the connection state issue, I had those issues well before I tried adding this.